lib/assets/data_table/main.js

```diff
@@ -116,7 +123,14 @@ export function init(ctx, data) {
       orderBy(newKey) {
         this.loading = true;
         const [key, order] = reorder(this.data.content.order_by, this.data.content.order, newKey)
-        ctx.pushEvent("order_by", { key, order });
+        const label = key ? this.data.content.columns[key].label : null
```

Comment:
Please let me know what I miss, but I think `content.columns` is an array, and `key` is a string, how does this work? Shouldn't we instead find a column with `column.key === key`?